firefly-docker.env

```diff
@@ -1,7 +1,11 @@
-#---- Set environment variables for the docker startup
-#---- Examples-
+##---- Set environment variables for the docker startup
+##---- Examples-
 #PROPS_FIREFLY_OPTIONS=$'{ "coverage":  {"hipsSourceURL" : "ivo://CDS/P/2MASS/color"} }'
 #ADMIN_PASSWORD=reset-me
 #CLEANUP_INTERVAL=3h
-#---- Empty declarations: environment will come from the shell
+
+## Use Tomcat Admin basic auth on protected resources.  Defaults to true.
```

Comment:
I would add a comment here that says what endpoint(s) is/are being protected and that clarifies that the intent here is to permit a deployer to take care of authorization externally (as Rubin will).

E.g. "Use Tomcat Admin basic auth on protected resources, including the Firefly administration endpoints under .../admin/*. Defaults to true.  May be disabled in deployments where authorization is handled externally, e.g., by applying authorization-dependent redirects in a Kubernetes environment."